dashboard/app/assets/javascripts/angular/testExecutor.service.js

```diff
@@ -25,7 +25,7 @@
           method: 'DELETE'
         }
       });
-      var testExecUpdateIntervalPeriod = 5; // seconds
+      var testExecUpdateIntervalPeriod = 1; // seconds
```

Comment:
@mDibyo Okay. It sounds like you've got a better understanding. I just wanted to confirm that this isn't going to be a performance problem in some scenario where many many tests are in place.